src/plopp/backends/matplotlib/interactive.py

```diff
@@ -26,7 +26,13 @@ def __init__(self, View, *args, **kwargs):
         super().__init__(
             [
                 self.top_bar,
-                HBox([self.left_bar, self._view.canvas.to_widget(), self.right_bar]),
+                HBox(
+                    [
+                        self.left_bar,
+                        VBox([self._view.canvas.to_widget()]),
```

Comment:
I don't understand what this does, why is the widget wrapped in a `VBox`?

Reply:
This was an annoying thing where the buttons would end up all the way to the right, and for some reason, wrapping it in a `VBox` helps.
However, I now see that's it's still a little broken (see #323).

Reply:
I think i fixed it.